### DISCUSSION TITLE
Add variable to not use Red Hat subscription

### DISCUSSION THREAD

**Discussion by nesies** [vmware/packer-examples-for-vsphere#975] [Ideas]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Description

Red Hat subscription is mandatory in kickstart and playbook.
But somebody (me) might want to not use subscription.

### Use Case(s)

a Red Hat without subscription

### Potential Configuration

add a variable _rhsm_enabled__ in _builds/rhsm.pkrvars.hcl.example_
And use it in packer / playbook

- ansible/roles/base/tasks/redhat.yml
- builds/linux/rhel/8/data/ks.pkrtpl.hcl
- builds/linux/rhel/8/linux-rhel.pkr.hcl
- builds/linux/rhel/8/variables.pkr.hcl
- builds/linux/rhel/9/data/ks.pkrtpl.hcl
- builds/linux/rhel/9/linux-rhel.pkr.hcl
- builds/linux/rhel/9/variables.pkr.hcl
- builds/rhsm.pkrvars.hcl.example

**Comment by tenthirtyam** [comment]
I'm please feel free to open a pull request for this enhancement. We will review and merge after the CLA is approved and all tests pass.
